test/testdata/generate.sh

```diff
@@ -132,14 +132,14 @@ scaffold_test_project project-v2 --project-version=2
 # [Currently, default CLI plugin] - Project version 3 (default) uses plugin go/v3 (default).
 scaffold_test_project project-v3
 scaffold_test_project project-v3-multigroup
-scaffold_test_project project-v3-addon-and-grafana --plugins="go/v3,declarative,grafana/v1-alpha"
+scaffold_test_project project-v3-addon-and-grafana --plugins="go/v3,declarative,multi-module,grafana/v1-alpha"
 scaffold_test_project project-v3-config --component-config
 scaffold_test_project project-v3-with-deploy-image
 
 # [Next version, alpha] Project version v4-alpha
 scaffold_test_project project-v4 --plugins="go/v4-alpha"
 scaffold_test_project project-v4-multigroup --plugins="go/v4-alpha"
-scaffold_test_project project-v4-addon-and-grafana --plugins="go/v4-alpha,declarative,grafana/v1-alpha"
+scaffold_test_project project-v4-addon-and-grafana --plugins="go/v4-alpha,declarative,multi-module,grafana/v1-alpha"
```

Comment:
@jakobmoellersap I think we need a new dirs/lines here 

scaffold_test_project project-v4-multi-module --plugins="go/v4-alpha,multi-module"

So that we can check the result of this one only

Also, we will need that for go/v3 and you need to change the above lines to generated the same apis and etc for this scenario

You can run make generate to get it all done.